packages/react-heartwood-components/src/components/Button/Button-story.js

```diff
@@ -30,7 +24,6 @@ stories.addDecorator(withKnobs)
 stories
 	.add('Primary', () => (
 		<Button
-			isSmall={isSmall}
```

Comment:
don't know if we should keep this prop in and replace as `isSmall={boolean('isSmall', false}` so it appears with the knobs in storybook? that might be why it was written this way. @jtini ?